sdcm/scylla_bench_thread.py

```diff
@@ -152,44 +137,54 @@ def verify_results(self):
 
         return sb_summary, errors
 
-    def _run_stress_bench(self, node, loader_idx, stress_cmd, node_list):
+    def _run_stress(self, loader, loader_idx, cpu_idx):
+        cpu_options = ""
+
+        if self.stress_num > 1:
+            cpu_options = f'--cpuset-cpus="{cpu_idx}"'
+
+        docker = RemoteDocker(loader, "scylladb/hydra-loaders:scylla-bench-v0.1.8",
+                              extra_docker_opts=f'{cpu_options} --label shell_marker={self.shell_marker}')
```

Comment:
Please, merge-in the last commit here. No need to split it.
If you want, you can add a note with description of "why" we need host networking.